packages/squiggle-lang/src/rescript/Utility/XYShape.res

```diff
@@ -453,6 +453,42 @@ module PointwiseCombination = {
     T.filterOkYs(newXs, newYs)->Ok
   }
 
+  // *Dead code*: Nuño wrote this function to try to increase precision, but it didn't work.
+  // If another traveler comes through with a similar idea, we hope this implementation will help them.
```

Comment:
You could still explain what this is supposed to do. "enrich" really doesn't mean much.